jujugui/static/gui/src/app/components/inspector/inspector.js

```diff
@@ -22,18 +22,49 @@ YUI.add('inspector-component', function() {
 
   juju.components.Inspector = React.createClass({
 
+
+    /**
+      Get the current state of the inspector.
+
+      @method _getInitialState
+      @returns {String} The current state.
+    */
+    _getInitialState: function() {
+      return {
+        activeComponent: this.props.getAppState(
+            'current', 'sectionA', 'component')
+      };
+    },
+
+    /**
+      Callback for when the header back is clicked.
+
+      @method _backCallback
+    */
+    _backCallback: function() {
+      var state = {
+        sectionA: {
+          component: 'services'
+        }
+      };
+      this.props.changeState(state);
+    },
+
     render: function() {
-      var title = 'mediawiki';
-      var type = 'uncommitted';
-      var count = 5;
+      var state = this._getInitialState();
+      var childComponent = '';
+      if (state.activeComponent === 'inspector') {
```

Comment:
`this.state.activeComponent` the `_getInitialState` is called by React automatically to populate the state object